src/test/java/com/facebook/openwifirrm/optimizers/TestUtils.java

```diff
@@ -65,11 +72,21 @@ public static JsonArray createDeviceStatusDualBand(int channel2G, int txPower2G,
 		return jsonList;
 	}
 
+	private static int channelToFrequency(int channel) {
```

Comment:
These two equations are actually taken from the IEEE 802.11 standard, I will add a comment mentioning that.

Reply:
Add a comment that the unit is MHz. Some folks like indicating that in the name itself -- e.g. `channelToFrequencyMHz`